From 3d0b78eb855dc3797e614a72019afbebbebc536a Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Mon, 28 Aug 2017 08:58:10 +0200
Subject: [PATCH] WaE: narrowing conversion

error C2398: Element '3': conversion from 'unsigned __int64' to 'unsigned long' requires a narrowing conversion
---
 src/lib/QXPContentCollector.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/lib/QXPContentCollector.cpp b/src/lib/QXPContentCollector.cpp
index 6a227a6..9c3076d 100644
--- a/src/lib/QXPContentCollector.cpp
+++ b/src/lib/QXPContentCollector.cpp
@@ -943,7 +943,9 @@ void QXPContentCollector::drawText(const std::shared_ptr<Text> &text, const Link
         break;
       }
 
-      const unsigned long spanTextEnd = std::min<unsigned long>({ charFormat.afterEndIndex(), paragraph.afterEndIndex(), text->text.length(), textEnd });
+      const auto spanTextEnd = static_cast<unsigned long>(
+                                 std::min<uint64_t>({ charFormat.afterEndIndex(), paragraph.afterEndIndex(), text->text.length(), textEnd })
+                               );
 
       if (charFormat.format->isControlChars)
       {
-- 
2.13.0

